scripts/Muon/GUI/Common/ADSHandler/muon_workspace_wrapper.py

```diff
@@ -142,7 +142,8 @@ def show(self, name=''):
                 self.add_directory_structure()
                 # Add to the appropriate group
                 group = self._directory_structure.split("/")[-1]
-                mtd[group].add(self._workspace_name)
+                if not mtd[group].__contains__(self._workspace_name):
```

Comment:
Should this be using the ADS instead of mtd?
```suggestion
                if not AnalysisDataService.retrieve(group).__contains__(self._workspace_name):
```